### ISSUE TITLE
[Proposal] Gymnasium/MuJoCo-v5 dataset generation

### ISSUE THREAD

**Issue by Kallinteris-Andreas** [Farama-Foundation/Minari#243]
### Proposal
I will make Datasets for the `gymnasium/mujoco-v5` environments
The Datasets will be provided in 5 stages for sake of review process

#### Stage 1
Environments: `InvertedPendulum`, `InvertedDoublePendulum`
Datasets: 1 "Medium" + 1 "Expert" (the "expert policy" here mean "perfect", because the environments are so simple)
#### Stage 2
Environments: `Reacher`, `Pusher`
Datasets: 1 "Medium" + 1 "Expert" 
#### Stage 3
Environments: `Hopper`, `Walker`, `HalfCheetah`
Datasets: 1 "Simple" + 1 "Medium" + 1 "Expert" 
#### Stage 4
Environments: `Swimmer` (this is a bit of special case cause of the learning rate requirement)
Datasets: 1 "Expert" 
#### Stage 5
Environments: `Ant`, `Humanoid`, `Humanoid Standup`
Datasets: 2 "Simple" + 2 "Medium" + 1 "Expert" (note: the expert here will be a fine-tuning policy, like we have Ant-v5 already) 
#### Stage 6 (Bonus maybe?)
Environments (from menagerie realistic robots):  ["GoogleBarkour-v1"](https://github.com/google-deepmind/mujoco_menagerie/blob/main/assets/google_barkour_vb-barkour_vb.png), ["Unitree-G1"](https://github.com/google-deepmind/mujoco_menagerie/blob/main/assets/unitree_g1-g1.png)
Datasets: (probably) 1 "Medium" + 1 "Expert" 

### Motivation
This is already well requested.

### Alternatives
Any suggestion welcome, or alterations to current plan are welcome

### Additional context
The Dataset generation process will begin after gym1.0 is released.
I am the project manager of Gymnasium Robotics

The Rodrigo's code (https://github.com/Farama-Foundation/minari-dataset-generation-scripts/pull/8/files) will likely be adapted for the new versions of SB3, GYM, MINARI, rather than writing it from scratch

### Checklist
- [] I have checked that there is no similar [issue](https://github.com/Farama-Foundation/Minari/issues) in the repo (**required**)

**Comment by younik** [comment]
Perfect, this sounds good, thanks!

I suggest to use `MuJoCo` as namespace for the datasets

**Comment by jamartinh** [comment]
Hi, please add the excluded variables of observations in infos please as in D4RL.

**Comment by Kallinteris-Andreas** [comment]
@jamartinh waiting on https://github.com/Farama-Foundation/Minari/pull/250

**Comment by Kallinteris-Andreas** [comment]
Example:
x- and y-coordinates of the Ant's torso (https://gymnasium.farama.org/environments/mujoco/ant/#observation-space)

He is referring to the fact that `info` is not being included in datasets

**Comment by jamartinh** [comment]
@younik 

With respect to  https://github.com/Farama-Foundation/Minari/pull/250

You have to decide, accept the proposal or reject it. I don't see any other option. I have proposed a solution and a default behavior. Or either reject the enhancement for the user in favor of design and expectation simplicity or adopt only the new format, so:

1. Keep the enhancement as optional and defaulting to current behavior https://github.com/Farama-Foundation/Minari/pull/250
2. Reject the enhancement   https://github.com/Farama-Foundation/Minari/pull/250
3. Adopt the enhancement https://github.com/Farama-Foundation/Minari/pull/250 as the new way for now on

This is up to you.

**Comment by younik** [comment]
> Example: x- and y-coordinates of the Ant's torso (https://gymnasium.farama.org/environments/mujoco/ant/#observation-space)
> 
> He is referring to the fact that `info` is not being included in datasets

Oh, got it, thanks!



> [@younik](https://github.com/younik)
> 
> With respect to [#250](https://github.com/Farama-Foundation/Minari/pull/250)
> 
> You have to decide, accept the proposal or reject it. I don't see any other option. I have proposed a solution and a default behavior. Or either reject the enhancement for the user in favor of design and expectation simplicity or adopt only the new format, so:
> 
>     1. Keep the enhancement as optional and defaulting to current behavior [Support heterogeneous dicts in infos #250](https://github.com/Farama-Foundation/Minari/pull/250)
> 
>     2. Reject the enhancement   [Support heterogeneous dicts in infos #250](https://github.com/Farama-Foundation/Minari/pull/250)
> 
>     3. Adopt the enhancement [Support heterogeneous dicts in infos #250](https://github.com/Farama-Foundation/Minari/pull/250) as the new way for now on
> 
> 
> This is up to you.

The fourth option is to implement it in a backward-compatible way, as I explained in the PR comment.
So for now, I will just reject https://github.com/Farama-Foundation/Minari/pull/250. I don't have an ETA for this feature, but maybe @gabrielemaraglino will have some bandwidth to work on it in the next months.
